.github/workflows/build-pr.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       contents: read
     strategy:
       matrix:
-        KUBERNETES_VERSION: ["1.24.6"]
+        KUBERNETES_VERSION: ["1.25.8", "1.26.3"]
```

Comment:
just curious why we can't use the same minor versions between Azure and regular e2e for 1.25?

Reply:
the latest stable versions of native k8s are 1.25 and 1.26, but 1.26 is a preview version in AKS, so I use 1.24 and 1.25 instead.

Reply:
make sense. I was actually referring to using 1.25.8 in regular e2e and 1.25.6 in azure e2e?

Reply:
Oh, I see. The reason is that they are the latest versions for each platform.

Reply:
ahh makes sense. thanks